File: ShopApp_Backend/Shopapp/src/main/java/com/project/Shopapp/dtos/FeedbackDTO.java
package com.project.Shopapp.dtos;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.project.Shopapp.models.Account;
import com.project.Shopapp.models.SanPham;
import jakarta.persistence.*;
import lombok.*;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
@Getter
@Setter
public class FeedbackDTO {
    @JsonProperty("user_id")
    private int USERID;

    @JsonProperty("content")
    private String NOIDUNG;

    private int SOSAO;

    @JsonProperty("product_id")
    private int MASANPHAM;
}

File: ShopApp_Frontend/src/app/dtos/feedback.dto.ts
// src/app/dtos/feedback.dto.ts
export class FeedbackDTO {
  user_id: number;
  content: string;
  sosao: number;
  product_id: number;

  constructor(data:any){
    this.user_id=data.userid;
    this.content=data.noidung;
    this.sosao=data.sosao;
    this.product_id=data.masanpham;
  }
}